### PR TITLE
add adafruit grand central m4 express to makecode

### PR DESCRIPTION
**Pull request by lalauria2000**
Hi, I want to add the adafruit grand central m4 express to makecode, I have added all the needed files as commits. Please let me know if it is possible to add it with all these files or if i need to change/upload something else. Thank you!

### CODE REVIEW

File: pxtarget.json [hunk 1]
```diff
@@ -113,13 +113,7 @@
         "aspectRatio": 1.0,
         "parts": true,
         "partsAspectRatio": 0.69,
-        "dynamicBoardDefinition": true,
```
Comment (pelikhan) [inline pxtarget.json:116]: this should not be deleted

File: pxtarget.json [hunk 2]
```diff
@@ -88,7 +89,6 @@
         "libs/sparkfun-samd21-mini-breakout",
         "libs/sparkfun-redboard-turbo",
         "libs/sparkfun-lumidrive",
-        "libs/brain-nrf52840",
```
Comment (pelikhan) [inline pxtarget.json:91]: did you forget to pull master?

**Comment by pelikhan** [comment]
Were you able to test it with a device? We don't have the grand central m4 in stock.

File: pxtarget.json (hunk 2 above)
Comment (lalauria2000) [reply pxtarget.json:91]: sorry what do you mean?

Comment (lalauria2000) [reply pxtarget.json:91]: I have not been able to test it with the device yet, I wanted to know if the files looked good before testing it

**Comment by pelikhan** [comment]
They look good but we haven’t tested the pin out at all.

________________________________

**Comment by lalauria2000** [comment]
Great then! I'll test it out with the device and get back to you shortly, thanks @pelikhan 

> They look good but we haven’t tested the pin out at all.
> ________________________________ From: lalauria2000 <notifications@github.com> Sent: Wednesday, December 16, 2020 12:16:39 AM To: microsoft/pxt-maker <pxt-maker@noreply.github.com> Cc: Peli de Halleux <jhalleux@microsoft.com>; Comment <comment@noreply.github.com> Subject: Re: [microsoft/pxt-maker] add adafruit grand central m4 express to makecode (#324) @lalauria2000 commented on this pull request.
> ________________________________ In pxtarget.json<https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fmicrosoft%2Fpxt-maker%2Fpull%2F324%23discussion_r543756978&data=04%7C01%7Cjhalleux%40microsoft.com%7Ccf87821466024c6a478008d8a14f7aa6%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C637436710017737583%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=LwpEZtLDbP%2BwO0edX4sEdvcIeCyCd1rGlRlPl%2BrcEH0%3D&reserved=0>:
> @@ -88,7 +89,6 @@
> "libs/sparkfun-samd21-mini-breakout", "libs/sparkfun-redboard-turbo", "libs/sparkfun-lumidrive", - "libs/brain-nrf52840", I have not been able to test it with the device yet, I wanted to know if the files looked good before testing it — You are receiving this because you commented. Reply to this email directly, view it on GitHub<https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fmicrosoft%2Fpxt-maker%2Fpull%2F324%23discussion_r543756978&data=04%7C01%7Cjhalleux%40microsoft.com%7Ccf87821466024c6a478008d8a14f7aa6%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C637436710017737583%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=LwpEZtLDbP%2BwO0edX4sEdvcIeCyCd1rGlRlPl%2BrcEH0%3D&reserved=0>, or unsubscribe<https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fnotifications%2Funsubscribe-auth%2FAA73QKLLTFXXUWJNITGC5ATSU7U5PANCNFSM4U432WLA&data=04%7C01%7Cjhalleux%40microsoft.com%7Ccf87821466024c6a478008d8a14f7aa6%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C637436710017747577%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=5RMmSwO79VDJK%2FRJAN%2BpckGNWaay0JmWri7q%2BzLcPc4%3D&reserved=0>.



> They look good but we haven’t tested the pin out at all.
> ________________________________ From: lalauria2000 <notifications@github.com> Sent: Wednesday, December 16, 2020 12:16:39 AM To: microsoft/pxt-maker <pxt-maker@noreply.github.com> Cc: Peli de Halleux <jhalleux@microsoft.com>; Comment <comment@noreply.github.com> Subject: Re: [microsoft/pxt-maker] add adafruit grand central m4 express to makecode (#324) @lalauria2000 commented on this pull request.
> ________________________________ In pxtarget.json<https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fmicrosoft%2Fpxt-maker%2Fpull%2F324%23discussion_r543756978&data=04%7C01%7Cjhalleux%40microsoft.com%7Ccf87821466024c6a478008d8a14f7aa6%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C637436710017737583%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=LwpEZtLDbP%2BwO0edX4sEdvcIeCyCd1rGlRlPl%2BrcEH0%3D&reserved=0>:
> @@ -88,7 +89,6 @@
> "libs/sparkfun-samd21-mini-breakout", "libs/sparkfun-redboard-turbo", "libs/sparkfun-lumidrive", - "libs/brain-nrf52840", I have not been able to test it with the device yet, I wanted to know if the files looked good before testing it — You are receiving this because you commented. Reply to this email directly, view it on GitHub<https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fmicrosoft%2Fpxt-maker%2Fpull%2F324%23discussion_r543756978&data=04%7C01%7Cjhalleux%40microsoft.com%7Ccf87821466024c6a478008d8a14f7aa6%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C637436710017737583%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=LwpEZtLDbP%2BwO0edX4sEdvcIeCyCd1rGlRlPl%2BrcEH0%3D&reserved=0>, or unsubscribe<https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fnotifications%2Funsubscribe-auth%2FAA73QKLLTFXXUWJNITGC5ATSU7U5PANCNFSM4U432WLA&data=04%7C01%7Cjhalleux%40microsoft.com%7Ccf87821466024c6a478008d8a14f7aa6%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C637436710017747577%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=5RMmSwO79VDJK%2FRJAN%2BpckGNWaay0JmWri7q%2BzLcPc4%3D&reserved=0>.

**Comment by pelikhan** [comment]
file name typo? devide.d.ts -> device.d.ts

**Comment by pelikhan** [comment]
Looks like all the files have been added so i'm closing this. Feel free to reopen if I was mistaken!

**Pull request closed without merging**